doc/script_commands.txt

```diff
@@ -1252,6 +1252,14 @@ This will allow you to visit 'Google' with the in-game browser using default dim
 
 Clicking 'Bing!' will open the in-game browser using the specified dimensions. (800x600)
 
+If you're using client from 2013-01-30 onwards, you can also use <ITEMLINK> to show
+the item's description. Gravity changed this into <ITEM> since 2015-07-29 onwards.
+
+    mes("Bring me an <ITEM>Apple<INFO>512</INFO></ITEM>.");
```

Comment:
hard to define this, many members told me its getting harder and harder to use older hex client
because the client translation project are always keep up to date with no backward compatibility

I have some older hexed client just simply because when I inactive, and active back
I always install newer stuffs, and the old stuffs just kept inside external harddisk

we should always encourage members to use latest client revision ... isn't it ?

and removing that line ... is just like hide this method from public

Reply:
No, I mean we should encourage them to use the `F_ItemMesInfo` and not `<ITEM>` or `<ITEMLINK>`.

Reply:
my opinion is show them both methods ...

ok 1 vote for showing 1 method (Emistry)
1 vote for showing both methods (AnnieRuru)

any one else vote for showing 1 method or showing both ?